packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec

```diff
@@ -69,14 +69,10 @@ cp -pa .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
-%{gem_instdir}/.codeclimate.yml
```

Comment:
I suggested to add `%exclude` to it. These files are still part of the gem but we don't want to ship these in the RPM. Same for `.ebert.yml`.